src/axom/core/Types.hpp

```diff
@@ -34,20 +50,21 @@ using uint16 = std::uint16_t; /*!< 16-bit unsigned integer type   */
 using int32 = std::int32_t;   /*!< 32-bit signed integer type     */
 using uint32 = std::uint32_t; /*!< 32-bit unsigned integer type   */
 
-// Note: KW -- We assume that AXOM_NO_INT64_T will be defined
-// on systems/compilers that do not support 64 bit integer types
-#ifndef AXOM_NO_INT64_T
+  // Note: KW -- We assume that AXOM_NO_INT64_T will be defined
+  // on systems/compilers that do not support 64 bit integer types
+  #ifndef AXOM_NO_INT64_T
```

Comment:
Is it safe to assume that all axom users have 64-bit types at this point?
If so, do we still need the CMake variable and compiler define associated with guarding against this?

Reply:
Our CMake doesn't seem to handle `AXOM_NO_INT64_T`.  Maybe it has been removed already.

Reply:
I think we should remove it in a follow-up PR.